packages/infra/lib/api-stack/fhir-converter-connector.ts

```diff
@@ -74,6 +74,7 @@ export function createQueueAndBucket({
     envType,
     alarmSnsAction,
     alarmMaxAgeOfOldestMessage: Duration.minutes(5),
+    maxMessageCountAlarmThreshold: 2000,
```

Comment:
I went with 2k here since we got a patient today with 1500 docs. We dont want to trigger the alarm as se initially add documents to the queue. Maybe should be lower or higher though kinda just a guess here